ipyleaflet/leaflet.py

```diff
@@ -1244,6 +1246,50 @@ def __init__(self, **kwargs):
         self.data = self._get_data()
 
 
+class WKTLayer(GeoJSON):
+    """WKTLayer class.
+
+    Layer created from a local WKT file or WKT string input.
+
+    Attributes
+    ----------
+    path: file path of local WKT file.
+    wkt_string: WKT string.
+    """
+
+    path = Unicode('')
+    wkt_string = Unicode('')
+
+    def __init__(self, **kwargs):
+        super(WKTLayer, self).__init__(**kwargs)
+        self.data = self._get_data()
+
+    @observe('path', 'wkt_string', 'style', 'style_callback')
+    def _update_data(self, change):
+        self.data = self._get_data()
+
+    def _get_data(self):
+        if self.path:
+            with open(self.path) as f:
+                wkt_data = f.read()
+                parsed_wkt = wkt.loads(wkt_data)
```

Comment:
Use `wkt.load(f)` instead?

Reply:
yes, this is a better way, I have updated